contracts/DomainRegistryV2.sol

```diff
@@ -0,0 +1,294 @@
+// SPDX-License-Identifier: UNLICENSED
+pragma solidity ^0.8.0;
+
+import "@openzeppelin/contracts-upgradeable/access/OwnableUpgradeable.sol";
+import "solidity-stringutils/src/strings.sol";
+import "hardhat/console.sol";
```

Comment:
// line 6 
// unnecessary import

Reply:
actually I have fixed it in the next homework after use **solhint** with **prettier**
if it's make sense

Reply:
done